doc/user_guide/electron_holography.rst

```diff
@@ -0,0 +1,37 @@
+Electron Holography
+*******************
+
+HyperSpy provides the user with two classes which can be used to process electron holography data:
+
+* :py:class:`~._signals.hologram_image.HologramImage`
+* :py:class:`~._signals.electron_wave_image.ElectronWaveImage`
+
+The usage of both classes is explained in the following sections.
+
+
+
+The ElectronWaveImage class
+===========================
+
+The :py:class:`~._signals.electron_wave_image.ElectronWaveImage` class can hold information about
+the complex electron wave. It inherits from :py:class:`~._signals.signal2d.Signal2D` and
+:py:class:`~._signals.complex_signal.ComplexSignal` and as such relevant properties like the
+`amplitude`, `phase` and the `real` and `imag` part of the complex data can be directly accessed
+and return appropriate :py:class:`~._signals.signal2d.Signal2D` signals.
+
+To transform a :py:class:`~._signals.signal2d.Signa2D` (or subclass) into a
+:py:class:`~._signals.electron_wave_image.ElectronWaveImage` use:
+
+.. code-block:: python
+
+    >>> im.set_signal_type('complex')

```

Comment:
Yes, my bad, I missed this during all of the renaming that was done. The text is also deprecated, as `ElectronWaveImage` does no longer inherit from `Signal2D`.